Sources/CasePathsMacros/CasePathableMacro.swift

```diff
@@ -257,6 +257,9 @@ extension EnumCaseElementListSyntax.Element {
         for index in associatedValue.parameters.indices {
           associatedValue.parameters[index].type.trailingTrivia = ""
           associatedValue.parameters[index].defaultValue = nil
+          if associatedValue.parameters[index].firstName?.tokenKind == .wildcard {
+            associatedValue.parameters[index].firstName = nil
```

Comment:
I tried to push to your branch but looks like your fork doesn't have those permissions.

I think wildcard parameters should not propagate the second name into the final tuple type, so wanna change this and re-record the snapshot?

```suggestion
            associatedValue.parameters[index].color = nil
            associatedValue.parameters[index].firstName = nil
            associatedValue.parameters[index].secondName = nil
```